packages/vscode-extension/src/extension.mts

```diff
@@ -129,6 +129,16 @@ export async function activate(context: vsc.ExtensionContext) {
 			)
 		}
 
+		context.subscriptions.push(
+			vsc.commands.registerCommand('spyglassmc.showOutput', () => {
```

Comment:
it'd be really nice if all these commands were refactored into an [enum](https://www.typescriptlang.org/docs/handbook/enums.html) so we aren't copy-pasting these strings.

```ts
enum VscodeCommand {
	ShowCacheRoot = 'showCacheRoot',
	ShowOutput = 'showOutput',
	// ...
}
```

can i make a PR that does this refactor?

Reply:
on second thought this may not be that useful since we need these strings in `packages/vscode-extension/package.json`. so we can't actually reference these enums.